integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParser.scala

```diff
@@ -91,6 +91,9 @@ class CarbonHelperSqlAstBuilder(conf: SQLConf,
       case Some(value) =>
         if (value.children.get(1).getText.equalsIgnoreCase("by")) {
           value.storageHandler().STRING().getSymbol.getText
+        } else if (value.children.get(1).getText.equalsIgnoreCase("as")
```

Comment:
can you call `value.children.get(1).getText` once and use the result

Reply:
ok, done